CONTRIBUTING.md

```diff
@@ -132,12 +132,28 @@ Review and replace command name and arguments depending on the command you are u
 
 ```
 
+### Contributing New Feature
 
-### API Interactions
+`Atlas CLI` and `MongoDB CLI` using[Cobra Framework](https://umarcor.github.io/cobra/) framework.
```

Comment:
should this be combined with https://github.com/mongodb/mongodb-atlas-cli/blob/cb185740535b25ee542c41c270b23d15ab405366/CONTRIBUTING.md#adding-a-new-command it seems to say many of the same things but with different words

Reply:
Fixed